Modules/Quotas/src/Quotas.Application/Infrastructure/Persistence/Repository/IQuotasRepository.cs

```diff
@@ -1,6 +1,8 @@
-﻿namespace Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+
+namespace Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
 
 public interface IQuotasRepository
 {
-
+    Task<IEnumerable<Quota>> FindQuotasByMetricKey(string metricKey, CancellationToken cancellationToken);
```

Comment:
This method is currently not used anywhere, as far as I can see. And it shouldn't be. In fact, the whole interface should be deleted.